bench/benchmark.exs

```diff
@@ -0,0 +1,22 @@
+sales_csv = "bench/data/10000_sales.csv"
+sales_string = File.read!(sales_csv)
+sales_list = String.split(sales_string, "\n")
+sales_stream = Stream.map(sales_list, & &1)
```

Comment:
```suggestion
sales_stream = sales_list |> Enum.map(& &1 <> "\n") |> Stream.map(& &1)
```

Maybe do a quick check to ensure they all return the same result? I think they currently don't.

Reply:
Currently this expression outputs `true`:

```ex
sales_stream = Stream.map(sales_list, & &1)

alias NimbleCSV.RFC4180, as: CSV
dbg(CSV.parse_enumerable(sales_list) == (CSV.parse_stream(sales_stream)) |> Enum.to_list())
```

So I don't think we need to add these newlines to the stream

Reply:
Ah, we treat the newlines in the stream as optional, perfect.